README.md

```diff
@@ -61,31 +61,37 @@ $ pipx install evg-module-manager
 
 ## Usage
 
-See the [usage](docs/usage.md) documentation for details about using this tool.
+See the [documentation](https://evergreen-ci.github.io/evg-module-manager/) for details about using this tool.
 
 ```bash
-evg-module-manager --help
 Usage: evg-module-manager [OPTIONS] COMMAND [ARGS]...
 
   Evergreen Module Manager is a tool help simplify the local workflows of evergreen modules.
 
 Options:
   --modules-dir PATH      Directory to store module repositories [default='..']
   --evg-config-file PATH  Path to file with evergreen auth configuration
-                          [default='/Users/user/.evergreen.yml']
+                          [default='/Users/username/.evergreen.yml']
   --evg-project TEXT      Name of Evergreen project [default='mongodb-mongo-master']
   --help                  Show this message and exit.
 
 Commands:
-  commit-queue  Submit changes from the base repository and any enabled modules to the...
-  create-branch Perform git checkout operation to create the branch.
-  disable       Disable the specified module in the current repo.
-  enable        Enable the specified module in the current repo.
-  git-commit    Perform git commit of changes in each module.
-  list-modules  List the modules available for the current repo.
-  patch         Create an Evergreen patch with changes from the base repo and any enabled...
-  pull-request   Create pull request for the changes in each module.
-  update-branch Perform git merge|rebase operation to update the branch.
+  disable            Disable the specified module in the current repo.
+  enable             Enable the specified module in the current repo.
+  evg-commit-queue   Submit changes from the base repository and any enabled modules to the...
+  evg-patch          Create an Evergreen patch with changes from the base repo and any...
+  git-add            Perform git add on all matching files in all modules.
+  git-branch-create  Create a new git branch on all enabled repositories.
+  git-branch-delete  Delete given branch in all enabled modules.
+  git-branch-pull    Pull the latest updates from remote repositories.
+  git-branch-show    Show existing branches on all enabled repositories.
+  git-branch-switch  Checkout the specified branch in all enabled modules.
+  git-branch-update  Get the latest changes from remote repositories and update the current...
+  git-commit         Get the git status of all modules.
+  git-restore        Perform git add on all matching files in all modules.
```

Comment:
What's the difference between git-restore and git-add? They seem to do the same thing based on the documentation

Reply:
fixed the docs, add and restore are opposites.